src/server.py

```diff
@@ -146,16 +149,14 @@ def run_results_day_flot_query():
                  'osx10.6',
                  'osx10.7',
                  'osx10.8']
-    db = create_db_connnection()
 
     data_platforms = {}
     for platform in platforms:
-        cursor = db.cursor()
-        cursor.execute("""select DATE(date) as day,sum(result="%s") as failures,count(*) as
-                          totals from testjobs where platform="%s" and date >= "%s" and date <= "%s"
-                          group by day""" % ('testfailed', platform, start_date, end_date))
-
-        query_results = cursor.fetchall()
+        query_results = session.query(Testjobs.date.label('day'),
+                                      func.count(Testjobs.result == 'testfailed'
+                                                 ).label("failures"),
+                                      func.count(Testjobs).label('totals')).filter(
+            and_(Testjobs.date >= start_date, Testjobs.date <= end_date)).group_by('day').all()
```

Comment:
hmm, I'm not sure exactly and we may need test for it. But when I use http://0.0.0.0:8157/data/results/flot/day/ to test it, I got:

<img width="1257" alt="screen shot 2016-08-08 at 9 22 04 pm" src="https://cloud.githubusercontent.com/assets/2166227/17481211/67390976-5dae-11e6-9e1a-eb818f6d1cde.png">
 And it looks the same with the living version on alertmanager server http://alertmanager.allizom.org/data/results/flot/day/